packages/core/test/types.test.ts

```diff
@@ -4552,4 +4553,17 @@ describe('snapshot methods', () => {
     snapshot.getMeta();
     snapshot.toJSON();
   });
+
+  // https://github.com/statelyai/xstate/issues/4931
+  it('fromPromise should not have issues with actors with emitted types', () => {
```

Comment:
describe blocks are evil 🤪 this should be moved outside of the current one because it has nothing to do with it